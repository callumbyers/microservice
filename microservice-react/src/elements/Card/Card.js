import React from 'react';
import { Card as DefaultCard } from '@material-ui/core';
import { withStyles } from '@material-ui/core/styles'

const styles = {
    default: {
        width: 400,
        height: 500,
        padding: 50
    }
};

const Card = (props) => {
    const { classes, children } = props;
    return (
        <DefaultCard className={classes.default}>
            {children}
        </DefaultCard>
    )
}


export default withStyles(styles)(Card);