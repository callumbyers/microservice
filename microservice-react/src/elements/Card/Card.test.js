import React from "react";
import renderer from "react-test-renderer";
import Card from "./Card";

test("renders correctly", () => {
  const card = renderer.create(<Card />).toJSON();
  expect(card).toMatchSnapshot();
});
