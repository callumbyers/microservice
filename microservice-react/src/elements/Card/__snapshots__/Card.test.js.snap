// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiPaper-root MuiCard-root Card-default-1 MuiPaper-elevation1 MuiPaper-rounded"
/>
`;
